inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cmd/shell/ShellTrackerImpl.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.service.cmd.shell;
+
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+public class ShellTrackerImpl implements ShellTracker {
+
+    private List<String> result = new ArrayList<String>();
+    private int code;
+    private Long processId;
+    private String runResult;
+
+    public int getCode() {
+        return code;
+    }
```

Comment:
remove getter and setter, since there is already have `@Data` annotation

Reply:
Done.